### PR TITLE
Revive image duplicates report & removal

### PR DESCRIPTION
**Pull request by frimpongopoku**
####  Summary / Highlights

After several manual tests, I think this  is ready to be tested. 
The actual deletion process here is muted. I have commented out that part of the code, so right now, we are only simulating deletion. All we need to do is uncomment the code when we are ready. 

[Link to Tech Spec](https://docs.google.com/document/d/1K90ERN1nO2nr84I6k7YBXGUuoCoMmML5Sg_x6BeF1xE/edit?usp=sharing) for some context

[How to test](https://docs.google.com/document/d/10PhaEVF8JCtguOUQkBqj3Tjpoa3GdwUZjTHGDGKSWC4/edit?usp=sharing)

[Here is what a final report looks like](https://drive.google.com/file/d/1kHVT9sC6be3kK2uHln8EuMWznMOyVs5q/view?usp=drive_link)

#### Requirements (place an `x` in each `[ ]`)
* [X] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [X] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [X] I've given my PR a meaningful title.
* [ ] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/api/handlers/media_library.py [hunk 1]
```diff
@@ -44,12 +44,13 @@ def read_image(self, request):
             return error
         return MassenergizeResponse(data=image_data)
 
-    @admins_only
+    # @admins_only
```
Comment (abdullai-t) [inline src/api/handlers/media_library.py:47]: is this intentional?

File: src/api/store/common.py [hunk 2]
```diff
@@ -216,21 +217,30 @@ def get_media_info(media):
...
-    )
-    print(duplicate_hashes)
-
-    if IS_LOCAL:
-        #check_hashes = Media.objects.exclude(hash__exact="").exclude(hash=None).values("hash").annotate(hash_count=Count("hash"))
-        check_hashes = Media.objects.exclude(hash__exact="").exclude(hash=None).values("hash","id").annotate(Count('hash'))
-        print(check_hashes, check_hashes.count())
+    if community_ids:
+        # First, filter the Media objects that belong to the given communities
+        print("Communities specified, duplicate context will be against only images in communities with ids : ",community_ids)
```
Comment (abdullai-t) [inline src/api/store/common.py:222]: please remove

File: src/api/store/common.py [hunk 3]
```diff
@@ -216,21 +217,30 @@ def get_media_info(media):
...
+            media_in_communities.exclude(hash__exact="")
+            .exclude(hash=None)
+            .exclude(hash=NON_EXISTENT_IMAGE)
+            .values("hash")
+            .annotate(hash_count=Count("hash"))
+            .filter(hash_count__gt=1)
+        )
+    else:
+        # If no community_ids are provided, check duplicates against all the media records
+        print("No communities specified, duplicate context will be against all images in the bucket....")
```
Comment (abdullai-t) [inline src/api/store/common.py:235]:
```suggestion
        logger.info("No communities specified, duplicate context will be against all images in the bucket....")
```

**Review by abdullai-t — APPROVED** [review]

**Review by archx3 — APPROVED** [review]

File: src/api/handlers/media_library.py (hunk 1 above)
Comment (BradHN1) [reply src/api/handlers/media_library.py:47]: I'd also like to know if this is intentional

**Pull request merged**